theme/templates/page.liquid

```diff
@@ -1,6 +1,20 @@
-{% assign has_black_background = page.metafields.accentuate.has_black_background %}
+{% assign variant = page.metafields.accentuate.page_styling_variant %}
+{% assign content_modules = page.metafields.accentuate.page_content.references %}
 
-<div class="Page site-padding-x {% if has_black_background %}Page--style-black {% endif %}">
+{{variant}}
+
+<div class="Page site-padding-x {% if variant == 'black-bg' %}Page--style-black-bg color-white bg-color-black {% else %} Page--style-white-bg {% endif %}">
   <h1 class="uppercase serif-mega">{{ page.title }}</h1>
-  <div class="Page__content pt2_5 pb2">{{ page.content }}</div>  
+    <div class="Page__content pt2_5 pb2 {% if variant == 'black-bg' %}col-12 md:col-9 {% endif %}">
+      {% for module in content_modules %}
+        {% assign referenced_fields = shop.metafields.acf_settings.global.fields | where: "section_name", module.section %}
+        {% assign section = module.section %}
+        {% case section %}
+        {% when 'page_markdown' %} 
+          {% render 'markdown' variant: variant classes: 'Markdown--style-block-content' markdown: shop.metafields.globals['page_markdown'][module.index].html %}  
```

Comment:
Out of curiosity, is this actually Markdown or is it a WYSWIG editor or something?

Reply:
If I log it, it shows up as strings like `##Headline 2 asdfasdfsdfsdf` so I think that's actually Markdown, right?

Reply:
Yuppers!